### PR TITLE
Add compat data for IDBIndex

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/API/IDBIndex

### CODE REVIEW

File: api/IDBIndex.json [hunk 1]
```diff
@@ -0,0 +1,1839 @@
...
+            "version_added": true
+          },
+          "chrome": [
+            {
+              "version_added": "24"
+            },
+            {
+              "version_added": "23",
+              "version_removed": "57",
+              "prefix": "webkit"
```
Comment (jpmedley) [inline api/IDBIndex.json]: Copy this information to webview_android and chrome_android.

File: api/IDBIndex.json [hunk 2]
```diff
@@ -0,0 +1,1839 @@
...
+            "chrome": [
+              {
+                "version_added": "24"
+              },
+              {
+                "version_added": "23",
+                "version_removed": "24",
+                "prefix": "webkit"
+              }
+            ],
```
Comment (jpmedley) [inline api/IDBIndex.json]: Copy this information to webview_android and chrome_android.

File: api/IDBIndex.json [hunk 3]
```diff
@@ -0,0 +1,1839 @@
...
```
Comment (jpmedley) [inline api/IDBIndex.json]: Copy this information to webview_android and chrome_android.

File: api/IDBIndex.json [hunk 4]
```diff
@@ -0,0 +1,1839 @@
...
+            },
+            "chrome": [
+              {
+                "version_added": "24"
+              },
+              {
+                "version_added": "23",
+                "version_removed": "24",
+                "prefix": "webkit"
+              }
```
Comment (jpmedley) [inline api/IDBIndex.json]: Copy this information to webview_android and chrome_android.

File: api/IDBIndex.json [hunk 5]
```diff
@@ -0,0 +1,1839 @@
...
+              "version_added": true
+            },
+            "chrome": [
+              {
+                "version_added": "24"
+              },
+              {
+                "version_added": "23",
+                "version_removed": "24",
+                "prefix": "webkit"
```
Comment (jpmedley) [inline api/IDBIndex.json]: Copy this information to webview_android and chrome_android.

File: api/IDBIndex.json [hunk 6]
```diff
@@ -0,0 +1,1839 @@
...
+              }
```
Comment (jpmedley) [inline api/IDBIndex.json]: Copy this information to webview_android and chrome_android.

File: api/IDBIndex.json [hunk 7]
```diff
@@ -0,0 +1,1839 @@
...
+            ],
```
Comment (jpmedley) [inline api/IDBIndex.json]: Copy this information to webview_android and chrome_android.

File: api/IDBIndex.json [hunk 8]
```diff
@@ -0,0 +1,1839 @@
...
```
Comment (jpmedley) [inline api/IDBIndex.json]: Copy this information to webview_android and chrome_android.

File: api/IDBIndex.json [hunk 9]
```diff
@@ -0,0 +1,1839 @@
...
+            },
+            "chrome": [
+              {
+                "version_added": "24"
+              },
+              {
+                "version_added": "23",
+                "version_removed": "24",
+                "prefix": "webkit"
+              }
```
Comment (jpmedley) [inline api/IDBIndex.json]: Copy this information to webview_android and chrome_android.

File: api/IDBIndex.json [hunk 10]
```diff
@@ -0,0 +1,1839 @@
...
+            ],
```
Comment (jpmedley) [inline api/IDBIndex.json]: Copy this information to webview_android and chrome_android.

**Comment by maboa** [comment]
That should be updated now :)

**Pull request merged**